### PR TITLE
Get cluster ids of all managed clusters for insights

### PR DESCRIPTION
**Pull request by zlayne**
**Related Issue:**  open-cluster-management/backlog#8621

### Description of changes
- [ ] After importing or creating a new OCP 4.x managed cluster, the user should see the latest CCX insights for the cluster.
- [ ] Hub cluster is included and shows CCX results when self-management is enabled.
- [ ] This is only applicable to those clusters which are OCP v4.x.  Any *ks or OCP 3.11 clusters should not have any policy report data associated with them.
- [ ] Keep insights in sync with the data in the CCX service (create & update - deletion of insights is done on cluster detach).

### CODE REVIEW

File: pkg/config/kubeClient.go [hunk 1]
```diff
@@ -53,17 +69,20 @@ func RESTClient(cfg *rest.Config) *rest.RESTClient {
...
+// Get the kubernetes dynamic client.
+func GetDynamicClient() dynamic.Interface {
+    mutex.Lock()
+    defer mutex.Unlock()
+    if dynamicClient != nil {
+        return dynamicClient
+    }
+    newDynamicClient, err := dynamic.NewForConfig(GetConfig())
+    if err != nil {
+        glog.Fatal("Cannot Construct Dynamic Client ", err)
```
Comment (xrajesh) [inline pkg/config/kubeClient.go:81]: Curious if we have to return nil .. please check

File: pkg/monitor/clustermonitor.go [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+		},
+		UpdateFunc: func(prev interface{}, next interface{}) {
+			m.processCluster(next, "update")
+		},
+		DeleteFunc: func(obj interface{}) {
+			m.processCluster(obj, "delete")
+		},
+	}
+
+	// Add Handlers to both Informers
```
Comment (xrajesh) [inline pkg/monitor/clustermonitor.go:62]: Fix the comment , we have one informer

File: pkg/config/kubeClient.go (hunk 1 above)
Comment (xrajesh) [reply pkg/config/kubeClient.go:81]: Please ignore , we call Fatal .

File: pkg/monitor/clustermonitor.go (hunk 2 above)
Comment (zlayne) [reply pkg/monitor/clustermonitor.go:62]: Will fix in another PR.

File: pkg/config/kubeClient.go (hunk 1 above)
Comment (zlayne) [reply pkg/config/kubeClient.go:81]: Using Fatal here - dont need a return

**Pull request merged**